test/suites/filemanip.sh

```diff
@@ -19,6 +19,19 @@ test_filemanip() {
   err=$(my_curl -o /dev/null -w "%{http_code}" -X GET "https://${LXD_ADDR}/1.0/containers/filemanip/files?path=/tmp/foo")
   [ "${err}" -eq "404" ]
 
+  myuid=1522341878
+  useradd --uid "${myuid}" myuser
```

Comment:
I'm a bit uncomfortable with modifying the system users in a permanent way.
Pretty much everything else in the test suite is either cleared on reboot or stored in the test run's temporary directory.

Did you explore the idea of using a container and copying the `lxc` CLI inside it with a `proxy` device of the LXD server's unix socket?

Reply:
> I just realized that it doesn't matter under which user the `lxc` is called, the only thing which matters is who owns the file which is being pushed.

Hah, indeed, great observation :)